### ISSUE TITLE
lambda and nested functions

### ISSUE THREAD

**Issue by wjaskowski** [zeehio/parmap#13]
Do you see any possibilities to allow lambda or nested functions with `parmap.map`?

**Comment by zeehio** [comment]
Not really. Unfortunately lambda functions are not pickable, so they are not suitable for multiprocessing.

There are some workarounds, see for instance: https://stackoverflow.com/questions/4827432/how-to-let-pool-map-take-a-lambda-function

I'll close this because I can't fix it, but feel free to comment if you have doubts.

**Comment by wjaskowski** [comment]
What about using https://github.com/pya/pickablelambda under the hood?

**Comment by zeehio** [comment]
Looks good! I don't have much free time right now to implement this. Feel free to submit a pull request if you want, otherwise I will do the work eventually :+1:
